File: ping-pong/client-java/src/test/java/net/corda/examples/pingpong/NodeDriver.java
package net.corda.examples.pingpong;

import net.corda.core.identity.CordaX500Name;
import net.corda.testing.driver.DriverParameters;
import net.corda.testing.driver.NodeParameters;
import net.corda.testing.node.User;

import java.util.Arrays;
import java.util.Collections;
import java.util.List;
import java.util.concurrent.ExecutionException;

import static net.corda.testing.driver.Driver.driver;

public class NodeDriver {
    public static void main(String[] args) {
        final List<User> rpcUsers = Arrays.asList(new User("user1", "test", Collections.singleton("ALL")));

        driver(new DriverParameters().withStartNodesInProcess(true).withWaitForAllNodesToFinish(true), dsl -> {

            try {
                dsl.startNode(new NodeParameters().withProvidedName(new CordaX500Name("PartyA", "London", "GB"))
                        .withRpcUsers(rpcUsers)).get();
                dsl.startNode(new NodeParameters()
                        .withProvidedName(new CordaX500Name("PartyB", "New York", "US"))
                        .withRpcUsers(rpcUsers)).get();
            } catch (InterruptedException | ExecutionException e) {
                e.printStackTrace();
            }

            return null;
        });


    }
}

File: implicit-cordapp-upgrades/workflows/v2-new-finality-flow/src/test/kotlin/net/corda/examples/obligation/workflow/NodeDriver.kt
package net.corda.examples.obligation.workflow

import net.corda.core.identity.CordaX500Name
import net.corda.core.utilities.getOrThrow
import net.corda.testing.driver.DriverParameters
import net.corda.testing.driver.driver
import net.corda.testing.node.TestCordapp
import net.corda.testing.node.User

fun main(args: Array<String>) {
    val user = User("user1", "test", permissions = setOf("ALL"))

    driver(DriverParameters(
            cordappsForAllNodes = listOf(
                    TestCordapp.findCordapp("net.corda.finance.contracts.asset"),
                    TestCordapp.findCordapp("net.corda.finance.schemas")),
            isDebug = true,
            startNodesInProcess = true,
            waitForAllNodesToFinish = true)) {
        val (nodeA, nodeB, nodeC) = listOf(
                startNode(providedName = CordaX500Name("PartyA", "London", "GB"), rpcUsers = listOf(user)),
                startNode(providedName = CordaX500Name("PartyB", "New York", "US"), rpcUsers = listOf(user)),
                startNode(providedName = CordaX500Name("PartyC", "Paris", "FR"), rpcUsers = listOf(user))
        ).map { it.getOrThrow() }

        startWebserver(nodeA)
        startWebserver(nodeB)
        startWebserver(nodeC)
    }
}